about.html

```diff
@@ -197,7 +197,7 @@ <h3 class="text-center mt-4 color-text">Partner</h3>
       </div>
     </footer>
   </main>
-  <script src="./js/main.js"></script>
+  <script src="./js/main.js" type="module"></script>
```

Comment:
- [ ]  In your about page for the mobile version as I can see the heading is very big I recommend to make the heading the same as the Given template since it is very important to stick with the given template as much as possible 🙏🏻♻✅


## Hint: Try to descrease the font size or the spacing around the heading to get the same given design

- Your design
![image](https://user-images.githubusercontent.com/58553711/211714857-d980598f-fb13-491e-92a4-dc28540c2924.png)


- Given design
![image](https://user-images.githubusercontent.com/58553711/211715039-775128d4-7f95-452a-9f2b-60ece6e8f4d0.png)